generate/templates/couchbase-server/Dockerfile.template

```diff
@@ -35,10 +67,10 @@ RUN set -x && \
     export INSTALL_DONT_START_SERVER=1 && \
     wget -N --no-verbose $CB_RELEASE_URL/$CB_PACKAGE && \
     echo "$CB_SHA256  $CB_PACKAGE" | sha256sum -c - && \
-    apt-get update && \
-    apt-get install -y ./$CB_PACKAGE && \
+    {{ .PKG_COMMAND }} update -y -q && \
+    {{ .PKG_COMMAND }} install -y ./$CB_PACKAGE && \
     rm -f ./$CB_PACKAGE && \
-    apt-get autoremove && apt-get clean && \
+    {{ .PKG_COMMAND }} autoremove && {{ .PKG_COMMAND }} clean all && \
```

Comment:
autoremove -y here too ... I think this only worked before because it happened that there wasn't anything necessary to uninstall.

Reply:
done.